testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -153,10 +143,16 @@ def _stop_docker_compose(self):
             logger.info("running %s" % cmd)
             subprocess.check_call(cmd, shell=True)
 
+    def _debug_log_containers_logs(self):
+        logs = self._docker_compose_cmd("logs --no-color")
+        for line in logs.split('\n'):
+            logger.debug(re.sub(r'[\r\n]*', '', line))
+
```

Comment:
As you are using the regular expression N times, one per row, it is more efficient to compile it.
You can define, at class level:

> re_newlines_sub = re.compile(r'[\r\n]*').sub

And then call it:

> logger.debug(self.re_newlines_sub('', line))

Reply:
Makes sense. Thanks for the tip.